### PR TITLE
[Filter] init properites

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Add init code - newly updated properties in filter.

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

**Comment by jaeyun-jung** [comment]
@anyj0527 YJ, plz check this PR.

**Review by anyj0527 — APPROVED** [review]
LGTM!

File: gst/nnstreamer/tensor_filter/tensor_filter_common.c [hunk 1]
```diff
@@ -546,6 +546,7 @@ gst_tensor_filter_properties_init (GstTensorFilterProperties * prop)
   prop->custom_properties = NULL;
   prop->accl_str = NULL;
   prop->hw_list = NULL;
+  prop->shared_tensor_filter_key = NULL;
```
Comment (myungjoo) [inline gst/nnstreamer/tensor_filter/tensor_filter_common.c:549]: Or... do memsetzero to *prop and set non-zero values only for the future safety?

Comment (jaeyun-jung) [reply gst/nnstreamer/tensor_filter/tensor_filter_common.c]: updated

**Review by myungjoo — APPROVED** [review]
LGTM

**Pull request merged**